docs/UserGuide.md

```diff
@@ -292,28 +302,35 @@ Format: `help`
 
 ### Importing flashcards : `import`
 
-Imports flashcards to LingoGO! using a CSV file.
-
-Format: `import CSV_FILE_PATH`
-
-* The file path can be absolute or relative to the location of the LingoGO! file.
-
 <div markdown="block" class="alert alert-info">
-**:information_source: Notes about `CSV_FILE_PATH`:**<br>
-* Users with no knowledge about file paths can just place the CSV file in the same folder that LingoGO! is located in
-  and fill the `CSV_FILE_PATH` parameter with the CSV file's name.
+**:information_source: Notes about importing CSV files made with Excel:**<br>
+* We advise against using Excel to create a CSV file to be imported into LingoGO!.
 </div>
 
+Imports flashcards from a CSV file and adds them to the existing list in LingoGO!
+(instead of replacing the current list)
+
+Format: `import CSV_FILE_NAME`
+
+* Place the CSV file that you wish to import in the *data* folder
+ (located in the same folder as the *lingogo.jar* file).
+* The CSV file must have headers "Language", "Foreign", "English".
```

Comment:
Should emphasize that the headers have to be exactly "Language,..." right now it doesn't seem clear enough and I would feel that "LANGUAGE" and "language" would also work as valid headers